html_css/intermediate_css/more-text-styles.md

```diff
@@ -49,7 +49,7 @@ body {
 Keep in mind that it's important to add a fallback font.  If you're linking to an external API, you have no guarantee that the URL won't change, or that the external API won't go down at some point. Having a reasonable fallback means that if something goes wrong, at least your site won't look completely broken.
 
 #### Downloaded Fonts
-It is also possible to use a font that you have downloaded from the web. In your CSS file, you import and define a custom font using the `@font-face` rule, and then use as you would any other font-family.
+It is also possible to use a font that you have downloaded from the web. In your CSS file, you import and define a custom font using the `@font-face` rule, and then use as you would any other font-family. As you can see this font-file extenstion is different than you may have seen elsewhere in the curriculum. There multiple different types of font file formats and can read more in depth about [here](https://fileinfo.com/filetypes/font). Please take care when choosing a font file format however, because some are not universally support by browsers. A list of browers and the font formats they support can be found [here](https://www.w3schools.com/css/css3_fonts.asp).
```

Comment:
Here are some changes:
1. `As you can see this font-file extenstion is different than you may have seen elsewhere in the curriculum. `

    I'm not confident about including this sentence, since this lesson might be the first time the curriculum mentions font file formats.

2. There is a typo and a grammatical error.

3. We could certainly improve the link text in this sentence to be more descriptive. "here" is not adequate.

What the paragraph could look like after the changes are applied:  

```suggestion
It is also possible to use a font that you have downloaded from the web. In your CSS file, you import and define a custom font using the `@font-face` rule, and then use as you would any other font-family. There are multiple types of font file formats and you can read more in depth about them on [fileinfo.com's page on Font File Formats](https://fileinfo.com/filetypes/font). Please take care when choosing a font file format however, because some are not universally supported by browsers. A list of browsers and the font formats they support can be found on [W3 Schools' page on CSS Web Fonts](https://www.w3schools.com/css/css3_fonts.asp).
```